pkg/restart/restart_test.go

```diff
@@ -50,23 +50,32 @@ func TestContext_Start(t *testing.T) {
 		cmd := "bad_command"
 		cc.cmd = exec.Command(cmd) // nolint:gosec
 
-		// TODO(nkryuchkov): Check if it works on Linux and Windows, if not then change the error text.
-		assert.EqualError(t, cc.Start(), `exec: "bad_command": executable file not found in $PATH`)
+		// TODO(nkryuchkov): Add error text for Windows
```

Comment:
Not related to this PR per se, but we do not need to check for windows compatibility.

Reply:
@jdknives I'm going to remove it